### DISCUSSION TITLE
希望可以增加一个直接用链接下载上传的功能

### DISCUSSION THREAD

**Discussion by BeanKaKa** [googidaddy/bili_upload_tool#14] [Ideas]
希望可以增加一个直接用链接下载上传的功能，不用去解析yt视频之类的

**Comment by googidaddy** [comment]
> 希望可以增加一个直接用链接下载上传的功能，不用去解析yt视频之类的

　意思上传不增加任何视频信息修饰，单纯下载并上传视频是吗？

**Reply by BeanKaKa** [reply]
是的，但是如果要增加的话可能会多出许多操作，比如转载来源，标题等的。因此这只是一个建议

**Reply by googidaddy** [reply]
> 

ok，可以增加。
